ui/src/app/cron-workflows/cron-workflow-list.tsx

```diff
@@ -156,7 +156,7 @@ export function CronWorkflowList({match, location, history}: RouteComponentProps
                                         <div className='columns small-2'>{w.metadata.namespace}</div>
                                         <div className='columns small-1'>{w.spec.timezone}</div>
                                         <div className='columns small-1'>
-                                            {w.spec.schedule != ''
+                                            {(w.spec.schedule ?? '') != ''
                                                 ? w.spec.schedule
                                                 : w.spec.schedules.map(schedule => (
```

Comment:
```suggestion
                                            {w.spec.schedule || w.spec.schedules.map(schedule => (
```

this specific one can be further simplified

Reply:
although tbh normalizing `schedule` to a single element array of `schedules` would be simpler to do everywhere